README.md

```diff
@@ -1,10 +1,22 @@
 ### Metabolic Atlas Map Viewer 3D
 
-This is a 3D graph viewer to visualize the metabolic atlas compound graph (metabolic network). See the [wiki](../../wiki) for more details.
+This is a 3D graph viewer to visualize the metabolic atlas compound graph (metabolic network), which is used as a self-contained feature of [Metabolic Atlas](https://metabolicatlas.org). The purpose of this library is to delvier the right technical solution that allows Metabolic Atlas to provide users with a useful experience while browsing and modifying a very large graph, with minimal knowledge required. The solution should be easy to work with and easy to maintain by programmers. The expectation is that most of the users of Metabolic Atlas will be biologists, followed by bioinformaticians; this means the 3D viewer should be minimalistic and intuitive. 
 
-Install dependencies with `npm install`, then run `npm run dev` to start the
-development server, or `npm run prepare` to build and minify the
-current version of the viewer.
+Below is a generic metabolic network example. Other graph structures are possible: no genes, duplicated metabolites.
+```
+Genes - Reaction - Metabolite - Reaction - Metabolite          
+             |            |  
+Reaction - Metabolite - Reaction - Metabolite
+  |  
+Genes  
+``` 
 
-The current version of the build is done with rollup, controlled by
-`rollup.config.js`, and will bundle the app with three-js.
+#### Installation
+
+Install dependencies with `npm install`, then run `npm run dev` to start the development server, or `npm run prepare` to build and minify the current version of the viewer.
+
+The current version of the build is done with rollup, controlled by `rollup.config.js`, and will bundle the app with three-js.
+
+#### Notes
+
+The library is automatically packaged and pushed to [npmjs.org](https://www.npmjs.com/package/@metabolicatlas/3d-network-viewer) via a GitHub Action. However, the action does not automatically bump the version, which means that occasional failures appear because of this.
```

Comment:
This last sentence is unclear to me and leads to the following questions. Does the version needs to be manually bumped? What should the developer do if failures occur?

Reply:
> Does the version need to be manually bumped?

I'm almost sure it needs to.

> What should the developer do if failures occur?

If build failures occur, the log would indicate if the version was not bumped.

Reply:
It would be good then to include info here about how and where the version should be bumped.

Reply:
Here's what I'm thinking:
```suggestion
The library is automatically packaged and pushed to [npmjs.org](https://www.npmjs.com/package/@metabolicatlas/3d-network-viewer) via a GitHub Action. However, the action does not automatically bump the version, which means that occasional failures appear because of this. To bump up the version, edit the corresponding field in `package.json` (see [this commit](https://github.com/MetabolicAtlas/3d-network-viewer/commit/35f0770b9643cc4adb52bc0b5c3f572be1609203) as an example).
```

Reply:
I think that's a good addition. Without context, I think one could read the previous sentences as: "during the process of automatically packaging and pushing to npmjs, failures may occur because the version does not automatically get bumped". Since I have also worked with this repo before, I might be overcompensating the need for clarity and context. Would be good to get more feedback on this from others.